src/sssom/util.py

```diff
@@ -80,6 +80,12 @@
 KEY_FEATURES = [SUBJECT_ID, PREDICATE_ID, OBJECT_ID, PREDICATE_MODIFIER]
 TRIPLES_IDS = [SUBJECT_ID, PREDICATE_ID, OBJECT_ID]
 
+# ! This will be unnecessary when pandas >= 3.0.0 is released
```

Comment:
Can you link an issue here that explains why it will be unnecessary? Just out of curiosity

Reply:
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.infer_objects.html#

Reply:
Cool if you could add that link to the comment in the script, that would be great

Reply:
Done.